telemetry-aware-scheduling/pkg/telemetryscheduler/telemetryscheduler.go

```diff
@@ -23,12 +23,13 @@ var tasPolicy = "telemetry-policy"
 
 //MetricsExtender holds information on the cache holding scheduling strategies and metrics.
 type MetricsExtender struct {
+	extender.NotBindingExtender
 	cache cache.Reader
 }
 
 //NewMetricsExtender returns a new metric Extender with the cache passed to it.
-func NewMetricsExtender(newCache cache.Reader) MetricsExtender {
-	return MetricsExtender{
+func NewMetricsExtender(newCache cache.Reader) *MetricsExtender {
```

Comment:
Was there a linting error that brought this about?

Reply:
I'll try to revert that change somehow

Reply:
success